### PR TITLE
Add Support for Windows Subsystem for Linux

### PR DESCRIPTION
**Pull request by nkgopaul**
Summary: This adds support for WSL by installing Mongo separately, starting an instance in a separate terminal, and running ce-platform with a separate environment variable.

Test:  Follow the instructions in the readme and run `npm start`. The project should start without any problems.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,22 @@ The Collective Experience (CE) Platform facilitates the creation and operation o
 4. Run `meteor npm install` to install local dependencies.
 5. Start the server using `meteor`.
 
+Hacky way in case meteor installation doesnt work (for Windows):
+
+apt-get install mongodb-server
```
Comment (youralien) [inline README.md]: @mtruong68 can you make this nicely markdown formatted? you might use the ` symbol to put things in nice font.
i.e. ```

File: README.md [hunk 2]
```diff
@@ -9,20 +9,23 @@ The Collective Experience (CE) Platform facilitates the creation and operation o
 5. Start the server using `meteor`.
 
 Hacky way in case meteor installation doesnt work (for Windows):
```
Comment (youralien) [inline README.md]: @mtruong68 forgot something: can you add a mention to the start script too? We're using `npm start` to do some of these commands.

**Comment by youralien** [comment]
@mtruong68 @nkgopaul when you're done with the PR, try to merge using the Github Web interface. I see some, but not all, your commits currently on master! That's not a clean merge to master in my mind!

**Comment by nkgopaul** [comment]
@mtruong68 I think you might have pushed your latest commit to master which resulted in all previous changes on this branch also being pushed to master. Anyways, I cleaned up the formatting a bit more and I think we're good to merge!

**Pull request merged**